UnityAds/UnityAdsAdapter/ALUnityAdsMediationAdapter.m

```diff
@@ -44,6 +44,10 @@ @interface ALUnityAdsMediationAdapter()
 @end
 
 @implementation ALUnityAdsMediationAdapter
+
+static NSString *const kMAKeyGameID = @"game_id";
+static NSString *const kMAKeySetMediationIdentifier = @"set_mediation_identifier";
```

Comment:
Where is this used?

Reply:
I'm not sure how it event got here, though it was from you guys. Removed the const that is not used in the code